src/components/input/test/SelectNext-test.js

```diff
@@ -21,13 +21,26 @@ describe('SelectNext', () => {
    *        Whether to renders SelectNext.Option children with callback notation.
    *        Used primarily to test and cover both branches.
    *        Defaults to true.
+   * @param {boolean} [options.defaultListboxAsPopover] -
+   *        Whether we should let the `listboxAsPopover` prop use default value
+   *        if not explicitly provided, or we should initialize it instead.
```

Comment:
Assuming most tests don't set this option, it means we'll be explicitly disabling the popover usage, right? We probably want tests to use the default behavior of the component except for tests that are specifically opting out.

Reply:
Yep, I agree. However, there are some tests failing when using the popover mode, mostly around testing that focusing works as expected in certain scenarios (first option gets focused when opened, the toggle recovers focus when closed, etc.). I assume it's related with the use of the top layer, but I couldn't confirm it.

I tried a couple of times to make them pass, but finally decided to disable the popover mode by default for the sake of getting existing tests to pass, and focus on popover mode only for new tests.

I will address this in a follow-up PR, as I think you are definitely right, but I wanted to offload some indirect complexity from this one.